learn/guides/organizing-ballerina-code.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: ballerina-organizing-code-left-nav-pages-swanlake
+title: Organizing Ballerina Code
+description: The sections below include information about packages, and how the growth of your source code can be managed.
+keywords: ballerina, programming language, ballerina packages, dependencies, importing modules
+permalink: /learn/organizing-ballerina-code/
+active: managing-dependencies
+intro: The sections below include information about packages, and how the growth of your source code can be managed.
+redirect_from:
+- /learn/user-guide/ballerina-packages/organizing-ballerina-code
+- /learn/user-guide/ballerina-packages/organizing-ballerina-code/
+- /learn/organizing-ballerina-code
+- /learn/user-guide/structuring-ballerina-code/
+- /learn/user-guide/structuring-ballerina-code
+- /learn/user-guide/ballerina-packages/
+- /learn/user-guide/ballerina-packages
+---
+
+## Package Structure
+
+Writing code in an organized manner from the beginning of the project is important for the lifecycle of the project and its maintainability in the long run. Organized code will make it easy to extend and improve your project over time. Ballerina project structure makes it easy to write clean code by eliminating repetitions, writing reusable code, adding new features without changing the existing code etc. To achieve this, Ballerina has the concept of Packages and modules. 
+
+Ballerina code is organized in a single shareable unit called a Package. Package is a collection of modules and a module is a collection of Ballerina source files, test files and resources. A package should contain at least one module which is called the default module. Each module has its own directory which is used to organize source files, test files and resources.
+
+A package is a collection of related modules that are versioned and distributed as a single unit. It is common in small projects to have only one module (default) in a package. As a result, the default module’s content is placed directly in the root of the package directory.
+
+`bal new` command creates the Package with the default module. This will generate the `Ballerina.toml` which identifies a directory as a package and will additionally generate a sample source file in the default module.
```

Comment:
```suggestion
he `bal new` command creates the package with the default module. This will generate the `Ballerina.toml` file, which identifies a directory as a package and will additionally generate a sample source file in the default module.
```